.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "openzwave-sys/open-zwave"]
+	path = openzwave-sys/open-zwave
+	url = https://github.com/OpenZWave/open-zwave.git
```

Comment:
This is super easy to update (and this is done like this in some similar rust crates (eg https://github.com/alexcrichton/git2-rs/tree/master/libgit2-sys, https://github.com/carllerche/curl-rust/tree/master/curl-sys) -- some other crates are copying though)